### DISCUSSION TITLE
Welcome to gem-push-notifications-concept Discussions!

### DISCUSSION THREAD

**Discussion by mfiebig** [gematik/gem-push-notifications-concept#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mabasger** [comment]
### Push Notifications in der TI
In diesem Forum finden Sie Informationen und Hilfe zu der Thematik "Push Notifications". Das Konzept ist TI-weit gültig und besteht grundlegend aus drei Komponenten:

1. der App, die Push Notifications erhalten soll (FdV)
2. dem Fachdienst, der fachliche Informationen verarbeitet und den Versicherten benachrichtigen soll
3. dem PushGateway, dass die tatsächliche Push Notification sendet

**App**
Die Schnittstelle zwischen FdV und Fachdienst wird mit folgenden UseCases geplant: De-/Registrierung für PushNotification, Listung und Registrierung zu Channels, Abruf von historischen Messages. 

**Fachdienst**
Implementiert zwei API - zum FdV (siehe oben) und zum PushGateway. Ferner ist die Verwaltung der registrierten Clients und deren abonnierten Channel Teil der Fachdienstimplementierung, sowie die fachlichen Trigger.

**PushGateway**
Eine standardisierte Komponente; beide offerierten API sind unabhängig von FdV und Fachdienst. Diese API ist zum einen die Push-Schnittstelle zu den Plattformen. Und zu dem Fachdienst / den Fachdiensten.
